app/scripts/modules/amazon/src/pipeline/stages/rollbackCluster/awsRollbackClusterStage.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const angular = require('angular');
+
+module.exports = angular.module('spinnaker.amazon.pipeline.stage.rollbackClusterStage', [
```

Comment:
Straight up borrowed ... what's the better way of doing this @anotherchrisberry? 

At first glance I didn't find any Reactified aws pipeline stages ... despite there being a fair bit of overlap.

Reply:
We haven't gotten around to React-ing pipeline stages yet...